src/vl53l0x_driver.cpp

```diff
@@ -103,18 +106,72 @@ int check_device_connection(int _file_descriptor) {
   return device_status.st_nlink;
 }
 
-void Start_Ranging(int i) {
-
-  VL53L0X_PerformSingleRangingMeasurement(pSensors[i],
-                                          &SensorsRangingMeasurementData[i]);
-  sensor_msg_array[i].proximity =
-      float(SensorsRangingMeasurementData[i].RangeMilliMeter) / 1000.0;
-  sensor_msg_array[i].header.stamp = ros::Time::now();
-  std::string frame = "sensor";
-  sensor_msg_array[i].header.frame_id = frame + std::to_string(i + 1);
-  sensor_msg_array[i].field_of_view = 0.436332;
-  sensor_msg_array[i].min_range = 0.03;
-  sensor_msg_array[i].max_range = 2.0;
+void thread1() {

```

Comment:
can you use a common method for the threads

Reply:
ok